app/bundles/LeadBundle/Controller/ListController.php

```diff
@@ -347,6 +347,8 @@ private function createSegmentModifyResponse(LeadList $segment, array $postActio
                             'objectId'     => $segment->getId(),
                         ]);
 
+                        $form = $segmentModel->createForm($segment, $this->get('form.factory'), $postActionVars['returnUrl']);
+
```

Comment:
You need to fix the php doc signature of Mautic\LeadBundle\Model\ListModel::createForm()
. to make phpstan happy at least:
```php
    /**
     * {@inheritdoc}
     *
     * @param       $entity
     * @param       $formFactory
     * @param array $options
     *
     * @return mixed
     *
     * @throws \Symfony\Component\HttpKernel\Exception\NotFoundHttpException
     */
    public function createForm($entity, $formFactory, ?string $action = null, $options = [])
    {
```